app/browser/api/ledger.js

```diff
@@ -2147,7 +2165,7 @@ const run = (state, delayTime) => {
 
   let winners
   const ballots = client.ballots()
-  const data = (synopsis) && (ballots > 0) && synopsisNormalizer(state)
+  const data = (synopsis) && (ballots > 0) && synopsisNormalizer(state, null, false, true)
```

Comment:
this method is border-line approaching too many arguments. I think if it needs any more logic, we should consider passing in a map instead of parameters.  Especially since some are optional

Reply:
good point